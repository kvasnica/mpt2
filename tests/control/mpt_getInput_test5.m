function mpt_getInput_test5

% tests that we inform the user that initial condition violates state
% constraints:
% http://control.ee.ethz.ch/~mpt/hg/mpt-25x?cs=1d9c35868c0a

Double_Integrator
sysStruct.xmax = [1;1];
sysStruct.xmin = [-1;-1];
ctrl = mpt_control(sysStruct, probStruct);
u = mpt_getInput(ctrl, [2;2]);
