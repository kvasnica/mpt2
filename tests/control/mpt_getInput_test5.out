Adding stabilizing target set constraint...
mpt_mpqp: 3 regions
-> Generated 1 partition.
Solution consists of 3 regions
State x = [2;2] violates state constraints!
MPT_GETINPUT: NO REGION FOUND FOR STATE x = [2;2]